src/database/initialization/source_to_json.sql

```diff
@@ -22,16 +23,23 @@ BEGIN
   FROM changemakers
   WHERE changemakers.id = source.changemaker_id;
 
-  RETURN jsonb_strip_nulls(jsonb_build_object(
-    'id', source.id,
-    'label', source.label,
-    'dataProviderShortCode', source.data_provider_short_code,
-    'dataProvider', data_provider_json,
-    'funderShortCode', source.funder_short_code,
-    'funder', funder_json,
-    'changemakerId', source.changemaker_id,
-    'changemaker', changemaker_json,
-    'createdAt', source.created_at
-  ));
+  -- The reason to go through the following hassle is we want null organization IDs returned.
+  -- At the same time, we want to omit non-existent changemakers, funders, and data providers.
+  SELECT jsonb_object_agg(name, value)
+  INTO source_json
+  FROM (VALUES
+    ('id', to_jsonb(source.id)),
+    ('label', to_jsonb(source.label)),
+    ('dataProviderShortCode', to_jsonb(source.data_provider_short_code)),
+    ('dataProvider', data_provider_json),
+    ('funderShortCode', to_jsonb(source.funder_short_code)),
+    ('funder', funder_json),
+    ('changemakerId', to_jsonb(source.changemaker_id)),
+    ('changemaker', changemaker_json),
+    ('createdAt', to_jsonb(source.created_at))
```

Comment:
Are these calls to `to_jsonb` needed to avoid mixing types?

```psql
pdc=> select jsonb_object_agg((values ('id', 3), ('createdAt', now()))); 
ERROR:  VALUES types integer and timestamp with time zone cannot be matched
LINE 1: ... jsonb_object_agg((values ('id', 3), ('createdAt', now())));
```

Reply:
@jasonaowen I guess. Again, thanks to some SO post here, it gave an error without and stopped giving an error with the call, so I kept the calls to `to_jsonb`.